Cargo.toml

```diff
@@ -48,9 +48,10 @@ debug = true
 name = "vector"
 section = "admin"
 maintainer-scripts = "distribution/debian/scripts/"
+conf-files = ["/etc/vector/vector.yaml", "/etc/default/vector"]
```

Comment:
We'll want to avoid adding this line as it triggers a `cargo-deb` bug and is unnecessary for files it `/etc`. See https://github.com/vectordotdev/vector/pull/18455